samples/subsys/ipc/openamp_rsc_table/src/main_remote.c

```diff
@@ -133,7 +133,7 @@ int mailbox_notify(void *priv, uint32_t id)
 	ARG_UNUSED(priv);
```

Comment:
Please add a commit message here and your `Signed-off-by` - it's mandatory - see [here](https://docs.zephyrproject.org/latest/contribute/guidelines.html#developer-certification-of-origin-dco).
You'll also make CI happy.

Otherwise, the PR LGTM.

Reply:
Thanks, added.